### DISCUSSION TITLE
Sprint 3

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#95] [Rencontres d'équipe]
### Notes suite à la rencontre avec Marcos

1. Barre de recherche
2. Validation champs formulaires
3. Modèles de voiture dans les formulaires
4. Information dans les listes
5.  Backlog
6. Présentation du 9 août
7. Zoom photo souhaitable

### Barre de recherche

- Diriger les résultats dans le catalogue
- Ajouter la barre de recherche à la page catalogue
- Ajouter la barre de recherche pour le côté admin

###  Validation des formulaires

- Ajouter la validation React avec message clair et bilingue

### Modèles de voiture dans les formulaires

- Ajouter le menu déroulant Constructeur
- Proposer les modèles de ce constructeur uniquement

### Information dans les listes

- Diminuer le nombre de colonne
- Au besoin, ajouter un lien vers une page Détails

### Backlog

- Ajouter une colonne Date échéance, ou créer un échéancier

### Présentation du 9 août

- 20 minutes ( l'Chemin' typique de chaque type d'utilisateur (Admin, Employé, Client))
- Petite intro - Démo du produit - petite conclusion
- Questions techniques
- Tester le projecteur au moins une journée avant!

**Comment by CollRen** [comment]
@claraquintela @NabilDjerroud Voici ci-haut le résumé des notes que j'ai prises lors de la renctre avec Marcos.

### Rigueur

Sinon, par rapport à nous pour le dernier Sprint, voici ce à quoi j'ai pensé par rapport à la rigueur:

**Tâhce terminée implique**

1. Traductions terminées
2. Liens fonctionnels
3. Code nettoyé et commenté
4. Numéro de tâche dans le commit
5. Inscrire le changement dans le Backlog

**Reply by CollRen** [reply]
@claraquintela Finalement, je pense que c'est mieux si tu fais un échéancier à part du Backlog... Plus global (Il peut ête dans un onglet du Backlog ou dans une discussion Git, ou comme tu veux... merci!

**Comment by claraquintela** [comment]
D'accord! je me concentre sur ça :)

Em ter., 16 de jul. de 2024 às 12:51, René de Montigny <
***@***.***> escreveu:

**Comment by CollRen** [comment]
@NabilDjerroud  Je crois que tout est en place afin que tu puisses travailler sur le paiement. Par exemple, toutes les tables et leurs associations, ou relations, sont prête.

Tiens-moi au courant, et demande-moi s'il manque des relations ou des infos.

**Reply by NabilDjerroud** [reply]
Parfait merci @CollRen , j'ai commencé déjà a travailler sur le panier, c'est correct pour le moment merci

**Comment by CollRen** [comment]
Lien de notre Drive: https://drive.google.com/drive/folders/1HZNzHpbrHSd_60XRLKrppwk806BGta-D?usp=sharing
